### ISSUE TITLE
Cannot undo with keyboard.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#35]
Originally reported on Google Code with ID 35

```
We should allow users to press "Ctrl+Z" to undo an action, rather than requiring them
to click on the "Undo" link.
```

Reported by `pcxunlimited` on 2010-09-12 05:08:49

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-12 05:15:30
- **Labels added**: Component-UndoBar

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-12 10:02:04
- **Status changed**: `Started`

**Comment by Pauan** [comment]
```
This issue was updated by revision cd598e2115.
```

Reported by `pcxunlimited` on 2010-09-12 10:15:21
- **Status changed**: `Pending`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-14 09:17:39
- **Labels added**: Accessibility

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-15 09:59:12
- **Labels added**: Type-Bug
- **Labels removed**: Type-Feature

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-16 13:20:10
- **Labels added**: Component-Undo

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-16 13:20:33
- **Labels removed**: Component-UndoBar

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2010-09-16 21:15:16
- **Status changed**: `Verified`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:44:47
- **Status changed**: `Fixed`
